app/src/main/java/org/jellyfin/androidtv/ui/playback/PlaybackController.java

```diff
@@ -115,6 +120,7 @@ public PlaybackController(List<BaseItemDto> items, CustomPlaybackOverlayFragment
         mFragment = fragment;
         mHandler = new Handler();
 
+        interactionTracker = lazyInteractionTracker.getValue();
```

Comment:
If you're gonna assign it directly on construct anyway you don't need to use a lazy injection. But that goes against the code style of all our existing Java code so I'd prefer not doing that.